common/src/main/java/gjum/minecraft/civ/snitchmod/common/Renderer.java

```diff
@@ -350,7 +364,7 @@ private static void renderTextFacingCamera(Component text, Vec3 pos, float offse
 		poseStack.mulPose(mc.gameRenderer.getMainCamera().rotation());
 		scale *= 0.005f * (mc.player.position().distanceTo(pos)/2.4);
 		scale = Utils.clamp(scale, 0.015f, 0.15f);
-		poseStack.scale(-scale, -scale, 1); // third component determines background distance
+		poseStack.scale(scale, -scale, 1); // third component determines background distance
```

Comment:
Any insights here? I remember trying different combinations.

Reply:
The text was flipped. Not sure what changed but it's `PoseStack.scale(x, y, z)` and it was flipped around y so I inverted x.